src/plugins/intel_gpu/src/kernel_selector/cl_kernels/cum_sum_partial_sum.cl

```diff
@@ -148,17 +153,15 @@ KERNEL(cum_sum_final)(
     PARTIAL_TYPE res = partial[ind];
 
     PARTIAL_TYPE sum = 0;
-    uint block_num = FUNC_CALL(get_block_num)(axes[AXIS]);
-    int n = 4;
-    for (int i = 0; i < block_num / n; ++i) {
-        unroll_for (int j = 0; j < n; ++j) {
```

Comment:
I suppose this internal loop added in sake of SIMD execution. Do you think it won't add performance benefit?